readme.txt

```diff
@@ -90,6 +90,7 @@ Gateways:
 * cpsms.dk
 * bulksmshyderabad.co.in
 * ozioma.net
+* Bulutfon.com
```

Comment:
Hello,

So, give me an access to your service provider to check it.

Regards